goss-testing/scripts/check_bgp_neighbors_established.sh

```diff
@@ -49,19 +49,46 @@ function err_exit
     exit $rc
 }
 
-kubectl -n metallb-system get cm metallb -o jsonpath='{.data.config}' > $TMPFILE ||
-    err_exit 10 "ERROR: Command failed (rc=$?): kubectl -n metallb-system get cm metallb -o jsonpath='{.data.config}' > $TMPFILE"
+TOKEN=$(curl -s -k -S -d grant_type=client_credentials -d client_id=admin-client -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
+
+# check if metalLB configmap exists
+kubectl -n metallb-system get cm metallb 2>&1 |grep -i error|wc -l > $TMPFILE ||
```

Comment:
When I run this on a 1.2 system, this the script fails on this line because the "grep -i error" returns 1.

And when I run this on a 1.0 system, the script fails on this line because the kubectl -n metallb-system get cm metallb 2>&1 part returns 1.

The set -o pipefail is what causes each fail and trigger the "err_exit" before it gets to the wc -l.

Reply:
It looks like lines 55 and 56 just shouldn't be there.

Reply:
revised testing items as suggested.  removed the error catch since we will have an error running this command on csm-1.0 system.